src/Symfony/Component/Debug/Tests/Fixtures/FinalClass.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Symfony\Component\Debug\Tests\Fixtures;
+
+/**
+ * @final since version 3.3.
+ */
+class FinalClass
```

Comment:
please add a test like
```php
/** @final since version 3.3 */
final class FinalClass {}
```

(or better, add test checking that no final declared class has final annotation)